compiler/ast/ast.nim

```diff
@@ -604,3 +606,39 @@ proc toHumanStr*(kind: TSymKind): string =
 proc toHumanStr*(kind: TTypeKind): string =
   ## strips leading `tk`
   result = toHumanStrImpl(kind, 2)
+
+
+proc setBaseFlags(n: PNode, base: NumericalBase) =
```

Comment:
Ask you base are belong to us.

Reply:
This should be a separate node kind for different literals